docs/algorithms/kem/ml_kem.md

```diff
@@ -7,9 +7,9 @@
 - **Authors' website**: https://pq-crystals.org/kyber/ and https://csrc.nist.gov/pubs/fips/203
 - **Specification version**: ML-KEM.
 - **Primary Source**<a name="primary-source"></a>:
-  - **Source**: https://github.com/pq-crystals/kyber/commit/10b478fc3cc4ff6215eb0b6a11bd758bf0929cbd with copy_from_upstream patches
+  - **Source**: https://github.com/pq-code-package/mlkem-native/commit/84398e7230fa31ba4241f5eb36bdc3c1dbbd5bcd
```

Comment:
How is this (no "copy_from_upstream patches" markup) possible with a file "mlkem-native.patch" present @bhess?

Reply:
Thanks for the catch! I didn't notice that the patch file name needs to contain the name of the algorithm "ml_kem" for the docs to pick up the "with copy_from_upstream patches". Updated the patch name now.